### PR TITLE
Replace @ember/polyfills assign with Object.assign

### PR DESCRIPTION
**Pull request by xomaczar**
Closes https://github.com/simplabs/ember-simple-auth/issues/2356

### CODE REVIEW

File: packages/ember-simple-auth/addon/authenticators/devise.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
 import { Promise } from 'rsvp';
 import { isEmpty } from '@ember/utils';
 import { run } from '@ember/runloop';
-import { merge, assign as emberAssign } from '@ember/polyfills';
+import { merge } from '@ember/polyfills';
```
Comment (marcoow) [inline packages/ember-simple-auth/addon/authenticators/devise.js:4]: Isn't `merge` the one that was [removed from `@ember/polyfills` in 4.0](https://api.emberjs.com/ember/4.0/modules/@ember%2Fpolyfills)?

File: packages/ember-simple-auth/addon/authenticators/torii.js [hunk 2]
```diff
@@ -73,7 +72,7 @@ export default BaseAuthenticator.extend({
       return this.get('torii').fetch(data.provider, data).then(
         (fetchedData) => {
           this._authenticateWithProvider(provider, fetchedData);
-          return emberAssign(data, fetchedData);
+          return Object.assign(data, fetchedData);
```
Comment (marcoow) [inline packages/ember-simple-auth/addon/authenticators/torii.js:75]: This won't work in IE11 (which I think ESA still supports as long as we support Ember 3.x).

**Review by marcoow — COMMENTED** [review]
Thanks for the effort – I'm not sure this is quite right yet though tbh…

**Comment by Windvis** [comment]
This seems like a duplicate of #2355?

File: packages/ember-simple-auth/addon/authenticators/torii.js (hunk 2 above)
Comment (bertdeblock) [reply packages/ember-simple-auth/addon/authenticators/torii.js:75]: Note that ESA already uses Object.assign here:
https://github.com/simplabs/ember-simple-auth/blob/master/packages/ember-simple-auth/addon/session-stores/cookie.js#L183

Comment (marcoow) [reply packages/ember-simple-auth/addon/authenticators/torii.js:75]: Ouch – thanks for pointing that out!

**Comment by marcoow** [comment]
indeed, it's a duplicate of #2355 – closing

**Pull request closed without merging**